trusted-execution-environments/README.md

```diff
@@ -0,0 +1,9 @@
+# Trusted Execution Environments
+
+## Intel SGX
+
+## AMD
```

Comment:
AMD and Risc-V are not TEEs and are not coherent with the other sections (`Intel SGX` and `ARM TZ`). Maybe add a section named "Solutions for Other Architectures / CPUs"

Reply:
No, they are not. For AMD I must find the name and add it, and Risc-V has multiple solutions (3, I think)